bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -142,17 +168,37 @@ public String getEncodedCurrentPagePath() {
     @Override
     public String getId() {
         if (getCurrentPage() != null) {
-            return ComponentUtils.getEncodedPath(getCurrentPage().getPath());
+            if (GuideWCMUtils.isForms(getCurrentPage().getPath())) {
```

Comment:
I see this code at multiple places. Can you create a utility for this and re-use it everywhere ?

Reply:
done